src/transferlistfilterswidget.cpp

```diff
@@ -359,7 +366,19 @@ void TransferListFiltersWidget::removeSelectedLabel() {
   // Save custom labels to remember it was deleted
   Preferences::instance()->removeTorrentLabel(label);
 }
-
```

Comment:
There should be one blank line between function definitions.
And in all similar places.